receiver/opencensusreceiver/config.go

```diff
@@ -77,7 +81,7 @@ type keepaliveEnforcementPolicy struct {
 func (rOpts *Config) buildOptions() (opts []Option, err error) {
 	tlsCredsOption, hasTLSCreds, err := rOpts.TLSCredentials.ToOpenCensusReceiverServerOption()
 	if err != nil {
-		return opts, fmt.Errorf("OpenCensus receiver TLS Credentials: %v", err)
+		return opts, fmt.Errorf("Error initializing OpenCensus receiver[%s] TLS Credentials: %v", rOpts.NameVal, err)
```

Comment:
Why the square brackets in the message? Do we use them normally in our error messages? I think we use %q to refer to entity names at least in config loading code.

Reply:
Square brackets- because that is how i've done in the past. The goal is to highlight the name of the receiver to help debugging purposes. It can be placed in any form of brackets or after a colon just want to make it obvious.
I don't remember the formatter '%q' is that for strings as well?

Reply:
I suggest to stick to what is already used elsewhere in the codebase (namely %q). Otherwise 1) you have to update every other place to the new style, 2) I do not see why square brackets are better than %q so it would be unnecessary extra work.

From https://golang.org/pkg/fmt/:
>%q	a single-quoted character literal safely escaped with Go syntax.

Reply:
Awesome - i didn't know about %q encapsulaiting the string with double quotes. That does achieve the same purpose to highlight the receiver name and I will use it for this case.